storage/database/db_manager.go

```diff
@@ -191,6 +202,9 @@ const (
 	databaseEntryTypeSize
 )
 
+const notInMigration = 0
+const inMigration = 1
```

Comment:
What about using `iota`?
https://yourbasic.org/golang/iota/

Reply:
Hmm? It would be better to use different names since `dbm.inMigration` is used. It is confusing.

Reply:
> What about using iota?

As there are only two variables, I thought it would be an overkill.

> Hmm? It would be better to use different names since dbm.inMigration is used. It is confusing.

I will change these names with the suffix `Flag`